Runtime/Model/Breadcrumbs/Storage/BacktraceStorageLogManager.cs

```diff
@@ -317,5 +323,22 @@ public double BreadcrumbId()
         {
             return _breadcrumbId;
         }
+
+        public string Archive()
+        {
+            if (!File.Exists(BreadcrumbsFilePath))
+            {
+                return string.Empty;
+            }
+
+            const string archivePattern = "{0}-1";
+            var copyPath = Path.Combine(_storagePath, string.Format(archivePattern, BreadcrumbLogFilePrefix));
+            if (File.Exists(copyPath))
+            {
+                File.Delete(copyPath);
+            }
+            File.Copy(BreadcrumbsFilePath, copyPath);
```

Comment:
I think `File.Copy` has an overload that supports overwriting the file:

https://docs.microsoft.com/pl-pl/dotnet/api/system.io.file.copy?view=netframework-2.0#system-io-file-copy(system-string-system-string-system-boolean)

It's supported even in .NET Framework 2.0, so you should be able to use it here.

Reply:
fixed. thx!